docs/reference/watcher/example-watches/watching-time-series-data.asciidoc

```diff
@@ -62,20 +62,20 @@ contain the words "error" or "problem".
 
 To set up the watch:
 
-. Define the watch trigger--a daily schedule that runs at 12:00 UTC:
+. Define the watch trigger--a daily schedule that runs at 12:00 Australian Eastern Standard Time (UTC+10:00):
 +
 [source,js]
 --------------------------------------------------
 "trigger" : {
   "schedule" : {
+    "timezone": "Australia/Brisbane",
     "daily" : { "at" : "12:00" }
   }
 }
 --------------------------------------------------
 +
-NOTE: In {watcher}, you specify times in UTC time. Don't forget to do the
-      conversion from your local time so the schedule triggers at the time
-      you intend.
+NOTE: In {watcher}, if the timezone is omitted then schedules default to UTC. `timezone` can be specified either
```

Comment:
Should this say "As of version 8.xx" or do we assume users match their docs version to the installed version?

Reply:
I don't think we usually put version information like that in the docs (although it would be useful sometimes).